src/Domain/Common/AuditableEntity.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+
+namespace Codidact.Domain.Common
+{
+    /// <summary>
+    /// Class that all of the entities inherit from
+    /// </summary>
+    public class AuditableEntity
+    {
+        /// <summary>
+        /// The date at which the entity was created
+        /// </summary>
+        public DateTime CreateDateAt { get; set; }
```

Comment:
I think we've misunderstood each other somewhere :)

Any creation date attribute should be called `CreatedAt` - we don't need both `Date` and `At` together.